### PR TITLE
detect_face bug-fix for list of torch tensors

### PR DESCRIPTION
**Pull request by mgp123**
bug-fix` .size `instead of `size()`.
`size` is the method. `size()` is the correct equivalent to `shape`

### CODE REVIEW

**Comment by mgp123** [comment]
Note that this would break `detect_face` for `PIL.Image` lists so this is a partial patch to be able to play with torch tensors. Needs more tweaks in order for this to work for all cases

**Comment by nighting0le01** [comment]
is that the only change required to make it work for batch of  torch.tensors? did you face any other issues like mismatch in mask shape

**Comment by mgp123** [comment]
Sorry. It was a while ago. I can't quite remember how I managed to do it.

File: models/utils/detect_face.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def detect_face(imgs, minsize, pnet, rnet, onet, threshold, factor, device):
     else:
         if not isinstance(imgs, (list, tuple)):
             imgs = [imgs]
-        if any(img.size != imgs[0].size for img in imgs):
+        if any(img.size() != imgs[0].size() for img in imgs):
```
Comment (timesler) [inline models/utils/detect_face.py:38]: Does this change break the ability to pass in lists of PIL images?
